addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1597,6 +1597,7 @@ const Wysiwyg = Widget.extend({
                     });
                     colorpicker.on('color_leave', null, ev => {
                         this.odooEditor.historyRevertCurrentStep();
+                        this.odooEditor._computeHistorySelection();
```

Comment:
Note for self:

The reason why I added that is by comparing hover+leaving and hover+hover quickly within the performance tool thing. And apparently the difference between them was that the onSelectionChange fired either between or after both events. And since this is the only permanent modification that concerns the function, I tried it and it works. The actual mechanisms is not yet understood.